job_templates/package_action_-_ansible_default.erb

```diff
@@ -35,11 +35,15 @@ model: JobTemplate
 # For Windows targets use the win_package module instead.
 ---
 - hosts: all
+  <% if input('pre_script').present? -%>
   pre_tasks:
     - shell: "<%= input('pre_script') %>"
+  <% end -%>
```

Comment:
actually this introduces a regression, the `-` at the end causes tasks not being well aligned. It's nested under hosts :-/

Reply:
the proper fix is to add `-` also to the begining so that it removes whitespaces before `<% end %>`, this is why it's better to always start <% at the beginning of the line and do the formatting inside erb, I'll send a fix in a minute

Reply:
see https://github.com/theforeman/community-templates/pull/469

Reply:
aw, sorry... I forgot how awful YAML is...

Reply:
I did think of that while looking at this PR, but not very well it appears.